addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/ahamodel/HeatingModel.java

```diff
@@ -29,6 +29,7 @@
         "nextchange" })
 public class HeatingModel {
     public static final BigDecimal TEMP_FACTOR = new BigDecimal("0.5");
+    public static final BigDecimal BIG_DECIMAL_TWO = new BigDecimal("2.0");
```

Comment:
Isn't this caused by the number of significant numbers, ie would its also work if you replace in the old code "0.5" by e.g. "0.500"?

Reply:
I'm not sure but I will test it and add feedback.

Reply:
I tested your proposal today but it results in the same error when we used "0.5".

Reply:
Pitty :-)

Reply:
Yes, the `BigDecimal` seems to be a big mess. ;-)